client/src/pages/SubmissionView.vue

```diff
@@ -0,0 +1,507 @@
+            elementum facilisis leo vel fringilla est ullamcorper. Arcu ac
+            tortor dignissim convallis aenean et tortor at risus. Tincidunt
+            augue interdum velit euismod in pellentesque massa placerat. Nisl
+            nunc mi ipsum faucibus. Eu feugiat pretium nibh ipsum. Donec et odio
+            pellentesque diam volutpat. Nunc sed velit dignissim sodales ut.
+            Venenatis urna cursus eget nunc scelerisque viverra mauris. Sem
+            viverra aliquet eget sit amet tellus. Magna fringilla urna porttitor
+            rhoncus dolor purus non enim. In nisl nisi scelerisque eu ultrices.
+            Tempor commodo ullamcorper a lacus vestibulum. Nisl nisi scelerisque
+            eu ultrices vitae auctor eu. Urna id volutpat
+            <span class="highlight"
+              >lacus laoreet non curabitur. Dolor magna eget est lorem ipsum
+              dolor. Mauris vitae ultricies leo integer malesuada nunc vel risus
+              commodo.</span
+            >
+          </p>
+          <h3>Commodo quis</h3>
+          <p>
+            Odio euismod lacinia at quis risus sed vulputate odio. Aliquam
+            eleifend mi in nulla. Ornare arcu odio ut sem nulla pharetra diam
+            sit amet. Nulla pharetra diam sit amet. Faucibus ornare suspendisse
+            sed nisi lacus sed. Commodo quis imperdiet massa tincidunt nunc
+            pulvinar sapien. Egestas tellus rutrum tellus pellentesque eu
+            tincidunt tortor. Pellentesque elit eget gravida cum sociis natoque.
+            Ut sem nulla pharetra diam sit amet. Sed lectus vestibulum mattis
+            ullamcorper. Sit amet nisl purus in mollis nunc sed id semper. Non
+            tellus orci ac auctor. In egestas erat imperdiet sed euismod.
+          </p>
+          <h3>Facilisi nullam vehicula ipsum a arcu</h3>
+          <p>
+            Blandit aliquam etiam erat velit scelerisque in dictum. Euismod quis
+            viverra nibh cras pulvinar mattis nunc sed blandit. Risus feugiat in
+            ante metus dictum at tempor. Facilisi nullam vehicula ipsum a arcu
+            cursus vitae congue mauris. Sit amet facilisis magna etiam tempor
+          </p>
+          <p>
+            orci eu lobortis. At quis risus sed vulputate odio ut enim blandit
+            volutpat. Tempor id eu nisl nunc mi. Malesuada nunc vel risus
+            commodo viverra maecenas accumsan lacus. Porttitor leo a diam
+            sollicitudin tempor. Blandit massa enim nec dui nunc mattis enim.
+            Elementum nisi quis eleifend quam adipiscing vitae proin sagittis.
+            Placerat orci nulla pellentesque dignissim. Condimentum id venenatis
+            a condimentum vitae sapien pellentesque.
+          </p>
+        </article>
+        <q-separator class="page-seperator" />
+        <section class="comments">
+          <div class="comments-wrapper">
+            <h3 class="text-h1">Overall Comments</h3>
+            <q-card>
```

Comment:
I would do the same thing I was suggesting with inlinecomments (single component, recursive (but single level bc we're not fully nesting comments ofc)

Reply:
This has been refactored as suggested.